### PR TITLE
(v9) Bump Go to 1.17.9

### PR DESCRIPTION
**Pull request by r0mant**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # The base image (buildbox:latest) is built by running `make -C build.assets`
 # from the base repo directory $GOPATH/gravitational.com/teleport
-FROM quay.io/gravitational/teleport-buildbox:go1.17.2
+FROM quay.io/gravitational/teleport-buildbox:teleport9
```
Comment (r0mant) [inline docker/Dockerfile:3]: @tcsc I changed it to use the new buildbox tag here, is this ok?

**Review by russjones — APPROVED** [review]

File: docker/Dockerfile (hunk 1 above)
Comment (tcsc) [reply docker/Dockerfile:3]: Not sure why it wasn't that already. Reasonably sure I put a PR in for it when I changed the rest.

**Review by tcsc — APPROVED** [review]

**Review by tcsc — APPROVED** [review]
Actually, the appropriate wat to change drone.yaml is to update `dronegen`

**Comment by r0mant** [comment]
> Actually, the appropriate wat to change drone.yaml is to update dronegen

@tcsc I did use dronegen. I'll double-check again before merging just in case.

**Review by espadolini — APPROVED** [review]
Reminder to update the `e` ref once gravitational/teleport.e#428 lands

**Pull request merged**